ocw/lib/EC2.py

```diff
@@ -190,27 +198,16 @@ def cleanup_all(self) -> None:
         if PCWConfig.getBoolean('cleanup/vpc_cleanup', self._namespace):
             self.cleanup_vpcs()
 
-    def delete_elastic_ips(self, region):
```

Comment:
Is it causing issues?

Reply:
with VPC deletion I never can say for 100% anything. The problem that VPC has plenty of different entities under control ( subnets , routing tables , routes , security groups , IPs , gateways and some others :) )  all they have some very complex relation between each other which I didn't manage fully realize. And if you trying to delete them in wrong order you will get exception. All changes which you see here is result of several days reading internet trigger different algorithms of deletion getting some exceptions and starting over again . 
So to answer you direct question **I don't know** but my gut feeling told me to drop it :D

Reply:
I see, OK :+1: